src/components/read-along-component/read-along.tsx

```diff
@@ -636,15 +774,33 @@ export class ReadAlongComponent {
   returnTranslation(word: string, lang: string): string {
```

Comment:
The following suggestion is not in the scope of this PR, however, I have suggestions on how this code can become neater:

First, have a type of available mandatory translations:

```typescript
type InterfaceLanguage = keyof Translation;

interface Translation {
  "en": string;
  "fr": string;
}
```

Make the `lang` parameter optional, and provide it with `this.language`:

```suggestion
  returnTranslation(word: string, lang?: InterfaceLanguage): string {
      if (lang === undefined) lang = this.language;
```

If you want to _enforce at compile-time_ that all text is has both an English and French translations, then turn the type of `let translations` to an object of translations:

```typescript
let translations: { [message: string]: Translation } = {
 /* ... */
};
```

Then, if you have an untranslated message, TypeScript will bark at you at compile-time!

![Screenshot of VSCode complaining about untranslated text](https://user-images.githubusercontent.com/2294397/127356044-aac9aa95-a613-4c88-a9cf-f184ebe8cf8f.png)